core/migrations/0022_fix_jsonext_type.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2.19 on 2023-06-01 11:45
+from django.conf import settings
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0021_set_managed_to_true'),
+    ]
+
+    operations = [
+        migrations.RunSQL('ALTER TABLE [tblOfficer] alter column [JsonExt] NVARCHAR(MAX)'
+                          if settings.MSSQL else
+                          'select 1'),
```

Comment:
If there is a need to reverse the migration, using 'select 1' for pssql will result in an error stating that it is an irreversible action.

Reply:
how can you have an empty reversible migration ?

I don't thinks we will ever reverse it

Reply:
It may be useful in the feature, when I was fixing the migration for pssql I encountered the issue with irreversible migrations. So it can be a good practice to add something like:
```
migrations.RunSQL('ALTER TABLE [tblOfficer] alter column [JsonExt] NVARCHAR(MAX)'
                          if settings.MSSQL else
                          'select 1',
                          reverse_sql='ALTER TABLE [tblOfficer] ALTER COLUMN [JsonExt] TEXT' if settings.MSSQL else
                          'select 1'),
    ]
```